frontend/src/lib/Utils.tsx

```diff
@@ -394,3 +395,13 @@ export async function decodeCompressedNodes(compressedNodes: string): Promise<ob
     });
   });
 }
+
+export function isFeatureEnabled(name: string): boolean {
+  const flags = localStorage.getItem('flags');
```

Comment:
Discuss offline, updated to write feature flag to local storage only when it doesn't exist, and read only once from local storage and use it as global variable for a tab. Also, catch exception to prevent failure on reading disk.